test/support/test_utils.rb

```diff
@@ -49,17 +49,6 @@ def applied_migrations(db_config = nil)
     end
   end
 
-  def clear_schema(db_config = nil)
-    if db_config
-      db_config.each_value do |config|
-        ActiveRecord::Base.establish_connection(**config)
-        clear_schema_call
-      end
-    else
-      clear_schema_call
```

Comment:
@VladislavSokov probably that was the reason of why `test/dummy_app/db/secondary_schema.rb` file gets updated in each commit. Can we make it so that `test/dummy_app/db/secondary_schema.rb` changed will never be commited?

Reply:
clear_schema(db_config = nil) is unused method.

`test/dummy_app/db/secondary_schema.rb` already added to gitignore, but I'll try to fix it